source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -42,4 +42,4 @@
 19 January 2022 Release
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-- Introduced :ref:`Flexible Sync (preview) <flexible-sync>`, which allows clients to sync data without the need for partition keys.
+- Introduced :ref:`Flexible Sync <flexible-sync>`, which allows clients to sync data without the need for partition keys.
```

Comment:
This should remain (preview), since it's release notes that date back to a time when it actually was preview.

Reply:
```suggestion
- Introduced :ref:`Flexible Sync (preview) <flexible-sync>`, which allows clients to sync data without the need for partition keys.
```